xbmc/Application.cpp

```diff
@@ -252,60 +252,55 @@ using KODI::MESSAGING::HELPERS::DialogResponse;
 //extern IDirectSoundRenderer* m_pAudioDecoder;
 CApplication::CApplication(void)
   : m_pPlayer(new CApplicationPlayer)
+  , m_iScreenSaveLock(0)
```

Comment:
i'd consider using class initialization instead of this nightmare init list. it's just asking for never-ending order issues later imo.